tasks/configure-activation-key.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: CFME | Subscribe Using Activation Key | Incluide Tasks for Installing Custom Facts
+  include_tasks: tasks/install-custom-facts.yml
+
+- name: CFME | Subscribe Using Activation Key | Set Satellite 6 Activation Key Name
+  set_fact:
+    sat6_activation_key: "{{ ansible_local['cfme']['sat6_activation_key'] }}"
```

Comment:
I dont understand this variable access `ansible_local['cfme']['sat6_activation_key']` which could just be my ignorance at the finer points of Ansible.

Reply:
@badnetmask can you fill me in?

Reply:
It means you can either pass the variable sat6_activation_key via your inventory, or via the playbook itself (like the cfme_version_product variable).

Reply:
@badnetmask why wouldn't you just reference it as `"{{ sat6_activation_key }}"`? Am i going crazy or doesn't `ansible_local['cfme']['sat6_activation_key']` mean something different then `{{ sat6_activation_key }}` which is what you are looking for in the `when` statements. To be fair I havn't tested this yet myself, but my brain is just trying to understand this bit.

Reply:
Check this: https://github.com/RedHatOfficial/ansible-redhat_cloudforms/blob/942ea2fc5cf715d36b5da62620e8812cff29096a/tasks/configure-repositories.yml#L11
The same kind of reference already exists in the code. I'm just trying to maintain the same usability method already created before.

Reply:
@badnetmask the reason that call is that way is because it is referencing a custom fact that is in the `cfme` hash. Your `sat6_actviation_key` is not part of that custom fact so is not in the `cfme` hash. Have you tried these changes against a cluster yet? As written I am pretty sure it will error as `ansible_local['cfme']['sat6_activation_key']` will return nil.

Reply:
It works just fine for me, because I have the variable defined in my inventory, I can just yank that ansible_local line out of the inventory to avoid problems.